public/docs/_examples/gettingstarted/ts/src/tsconfig.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
Okay, but then I would strongly suggest we remove --noImplicitAny from
tsconfig.json. This goes against what Martin and I are planning for our
talk (you can see the dry run tomorrow...)

On Tue, Oct 13, 2015 at 1:21 PM John Papa <notifications@github.com> wrote:

> In public/docs/_examples/gettingstarted/ts/src/tsconfig.json
> <https://github.com/angular/angular.io/pull/253#discussion_r41919025>:
>
> > @@ -1,11 +0,0 @@
> > -{
>
> Right, for the reasons you point out I think its worth in getting started,
> just to give it to them. GS is about getting moving fat and getting early
> positive reinforcement. I think its better for now to give it to them.
>
> Otherwise its as you described:
>
>    - go to the folder
>    - init
>    - change settings
>    - go back a folder
>
> that is friction i would rather avoid in GS.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/angular.io/pull/253/files#r41919025>.
>